src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURE_SIZE = 6;
+    private static final int NUMBER_OF_DEFAULT_FIGURES = 3;
```

Comment:
let's use only one constant because number of default figures is always half of the total number of figures